roles/pulp_webserver/meta/main.yml

```diff
@@ -9,7 +9,7 @@ galaxy_info:
   platforms:
     - name: Debian
       versions:
-        - buster
+        - bullseye
```

Comment:
if you grep for "buster" in the "roles" directory, you'll see many other references.